tenant/tenant/main.go

```diff
@@ -45,7 +45,9 @@ func main() {
 		return
 	}
 
+	cred := common.MakeCredentialFromArgs(username, password)
```

Comment:
Where does 'username' and 'password' come from here? I don't see it being extracted from the command line?